scripts/carDemoGen.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+if [ -z "$1" ]
+  then
+    echo "carDemoGen.sh folder"
+    exit 1
+fi
+printf "http://127.0.0.1:6001" > $1/server.url
+mkdir $1/txs
+
```

Comment:
why the car demo is part of the server?

Reply:
Actually, it is only end2end example we have now. I had discussion with Yoav and he suggested make docker image be able to support car demo out of the box. But I will move this script to sdk.